INSTRUCTIONSmd

```diff
@@ -0,0 +1,57 @@
+# Renaissance Fair Dashboard
```

Comment:
I think this file name needs to have a dot in it

Reply:
You're right! Good catch!

Reply:
Oh we haven't incorporated the navlinks in yet. The links on the main card buttons are the same.

Reply:
We can make those links functional for consistency in the meantime!